portal/templates/gil/base.html

```diff
@@ -461,19 +461,19 @@ <h4 class="box-modal__title">Session Timed Out</h4><p class="box-modal__copy">Yo
                       function(data) {
                         if (data) {
                           if (data.name) {
-                            $("#access_code_info").html("Thank you, your access code '" + $("#shortcut_alias").val() + "' indicates you are located at the " + data.name + ". Proceeding to registration . . .");
+                            $("#access_code_info").html("{%trans%}Thank you, your access code '" + $("#shortcut_alias").val() + "' indicates you are located at the " + data.name + ". Proceeding to registration . . .{%endtrans%}");
```

Comment:
Huh, maybe it's an escaping issue with the quotes that's preventing it from being recognized as JS? Not a big deal, just would be nice to hide these details from translators if possible.